ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,902 @@
+
+#. Verify the ACLs were configured by running the following command:
+
+   .. code-block:: bash
+
+      ccloud kafka acl list --service-account 104349
+
+   .. code-block:: text
+
+         ServiceAccountId | Permission | Operation | Resource |    Name    |   Type
+       +------------------+------------+-----------+----------+------------+----------+
+         User:104349      | ALLOW      | WRITE     | TOPIC    | demo-topic | PREFIXED
+         User:104349      | ALLOW      | CREATE    | TOPIC    | demo-topic | PREFIXED
+
+#. Run the Java producer to ``demo-topic-2`` to prefix the ACLs:
+
+   .. code-block:: bash
+
+      mvn -q -f ../../clients/cloud/java/pom.xml exec:java -Dexec.mainClass="io.confluent.examples.clients.cloud.ProducerExample" -Dexec.args="/tmp/client.config demo-topic-2" -Dlog4j.configuration=file:log4j.properties > /tmp/log.3 2>&1
+
+#. Verify you see the ``10 messages were produced to topic`` message in the logs as
+   shown in the following example:
+
+   .. code-block:: text
+
+      PASS
+      [2020-08-29 13:52:39,012] WARN The configuration 'sasl.jaas.config' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
+      [2020-08-29 13:52:39,013] WARN The configuration 'ssl.endpoint.identification.algorithm' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
+      Producing record: alice	{"count":0}
+      Producing record: alice	{"count":1}
+      Producing record: alice	{"count":2}
+      Producing record: alice	{"count":3}
+      Producing record: alice	{"count":4}
+      Producing record: alice	{"count":5}
+      Producing record: alice	{"count":6}
+      Producing record: alice	{"count":7}
+      Producing record: alice	{"count":8}
+      Producing record: alice	{"count":9}
+      Produced record to topic demo-topic-2 partition [3] @ offset 0
+      Produced record to topic demo-topic-2 partition [3] @ offset 1
+      Produced record to topic demo-topic-2 partition [3] @ offset 2
+      Produced record to topic demo-topic-2 partition [3] @ offset 3
+      Produced record to topic demo-topic-2 partition [3] @ offset 4
+      Produced record to topic demo-topic-2 partition [3] @ offset 5
+      Produced record to topic demo-topic-2 partition [3] @ offset 6
+      Produced record to topic demo-topic-2 partition [3] @ offset 7
+      Produced record to topic demo-topic-2 partition [3] @ offset 8
+      Produced record to topic demo-topic-2 partition [3] @ offset 9
+      10 messages were produced to topic demo-topic-2
+
+#. Run the following commands to delete ACLs:
+
+   .. code-block:: bash
+
+      ccloud kafka acl delete --allow --service-account 104349 --operation CREATE --topic demo-topic --prefix
+      ccloud kafka acl delete --allow --service-account 104349 --operation WRITE --topic demo-topic --prefix
+
+   You should see a ``Deleted ACLs`` message.
+
+#. Create a new Kafka topic ``demo-topic-3``:
```

Comment:
This should be moved to the next section

Reply:
updated